_posts/2022-04-07-launching-open-source-operator-for-openSearch.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Introducing open source Kubernetes Operator for OpenSearch"
+authors:
+  - Opster
+date: 2022-04-07
+categories:
+  - technical-post
+twittercard:
+  description: "The Kubernetes OpenSearch Operator is used for automating the deployment, provisioning, management, and orchestration of OpenSearch clusters and OpenSearch dashboards."
+---
+
```

Comment:
![Screen Shot 2022-04-20 at 9 24 58 AM](https://user-images.githubusercontent.com/4873623/164280438-afa2af38-f5b8-49c2-8907-298e884e54dd.png)

Please add a short excerpt above 'Overview' so we see that on the front page